databricks/koalas/config.py

```diff
@@ -170,6 +170,16 @@ def validate(self, v: Any) -> None:
             lambda v: v in ('sequence', 'distributed', 'distributed-sequence'),
             "Index type should be one of 'sequence', 'distributed', 'distributed-sequence'.")),
 
+    Option(
+        key='compute.ordered_head',
```

Comment:
We could fix the doc together.

https://github.com/databricks/koalas/blob/defeaffef0b0527c821c9332ba5043a4db04d653/databricks/koalas/config.py#L110-L114